File: Day_28/Q110.java
import java.util.*;

//Class For Bank Account
class Bank{
    int accNo;
    String name;
    float balance;
}

class Q110{

    static Scanner sc = new Scanner(System.in);

    //Create New Account
    static void createAccount(ArrayList<Bank> acc){

        Bank b = new Bank();

        System.out.print("Enter Account Number : ");
        b.accNo = sc.nextInt();

        sc.nextLine();

        System.out.print("Enter Name : ");
        b.name = sc.nextLine();

        System.out.print("Enter Balance : ");
        b.balance = sc.nextFloat();

        acc.add(b);

        System.out.println("Account Created Successfully");
    }

    //Deposit Money In Account
    static void depositMoney(ArrayList<Bank> acc){

        int no;
        float amount;

        System.out.print("Enter Account Number : ");
        no = sc.nextInt();

        System.out.print("Enter Amount : ");
        amount = sc.nextFloat();

        for(int i = 0;i < acc.size();i++){

            if(acc.get(i).accNo == no){

                acc.get(i).balance += amount;

                System.out.println("Money Deposited");
                return;
            }
        }

        System.out.println("Account Not Found");
    }

    //Display All Accounts
    static void displayAccounts(ArrayList<Bank> acc){

        if(acc.size() == 0){
            System.out.println("No Account Available");
            return;
        }

        for(int i = 0;i < acc.size();i++){

            System.out.println("\nAccount Number : " + acc.get(i).accNo);
            System.out.println("Name : " + acc.get(i).name);
            System.out.println("Balance : " + acc.get(i).balance);
        }
    }

    public static void main(String args[]){

        ArrayList<Bank> acc = new ArrayList<>();

        int choice;

        do{

            //Menu Driven Program
            System.out.println("\n1. Create Account");
            System.out.println("2. Deposit Money");
            System.out.println("3. Display Accounts");
            System.out.println("4. Exit");

            System.out.print("Enter Choice : ");
            choice = sc.nextInt();

            if(choice == 1) createAccount(acc);

            else if(choice == 2) depositMoney(acc);

            else if(choice == 3) displayAccounts(acc);

        }while(choice != 4);

        sc.close();
    }
}

File: Day_28/Q110.cpp
#include <bits/stdc++.h>
using namespace std;

//Structure For Bank Account
struct Bank{
    int accNo;
    string name;
    float balance;
};

//Create New Account
void createAccount(vector<Bank> &acc){

    Bank b;

    cout << "Enter Account Number : ";
    cin >> b.accNo;

    cin.ignore();

    cout << "Enter Name : ";
    getline(cin,b.name);

    cout << "Enter Balance : ";
    cin >> b.balance;

    acc.push_back(b);

    cout << "Account Created Successfully\n";
}

//Deposit Money In Account
void depositMoney(vector<Bank> &acc){

    int no;
    float amount;

    cout << "Enter Account Number : ";
    cin >> no;

    cout << "Enter Amount : ";
    cin >> amount;

    for(int i = 0;i < acc.size();i++){

        if(acc[i].accNo == no){

            acc[i].balance += amount;

            cout << "Money Deposited\n";
            return;
        }
    }

    cout << "Account Not Found\n";
}

//Display All Accounts
void displayAccounts(vector<Bank> acc){

    if(acc.size() == 0){
        cout << "No Account Available\n";
        return;
    }

    for(int i = 0;i < acc.size();i++){

        cout << "\nAccount Number : " << acc[i].accNo << endl;
        cout << "Name : " << acc[i].name << endl;
        cout << "Balance : " << acc[i].balance << endl;
    }
}

int main(){

    vector<Bank> acc;
    int choice;

    do{

        //Menu Driven Program
        cout << "\n1. Create Account";
        cout << "\n2. Deposit Money";
        cout << "\n3. Display Accounts";
        cout << "\n4. Exit";
        cout << "\nEnter Choice : ";
        cin >> choice;

        if(choice == 1) createAccount(acc);

        else if(choice == 2) depositMoney(acc);

        else if(choice == 3) displayAccounts(acc);

    }while(choice != 4);

    return 0;
}
